### DISCUSSION TITLE
Lien blog/nos cantines

### DISCUSSION THREAD

**Discussion by gauthierandre** [betagouv/ma-cantine#1156] [Idées]
Les articles du blog restent relativement consultés, la page "nos cantines" est définitivement la plus visitée du site.
Ne serait-il pas intéressé, pour compléter l'information aux usagers et la valorisation des actions ainsi que de nos articles de faire apparaitre sur la carte d'une cantine, une vignette avec l'article qui a été rédigé à son sujet quand c'est le cas ?

**Comment by hfroot** [comment]
J'aime l'idée. Je vois pas des articles dans le blog où l'auteur n'est pas dans notre équipe. Quels articles ont été écrits par un.e gestionnaire ?

**Reply by gauthierandre** [reply]
Ah oui, c'est vrai ! C'est au nom des bizdev, en mettant en lumière une cantine. Peut-être qu'on peut identifier l'user de la cantine (et la cantine?) quelque part sur le blog, ou en admin, en rajoutant un choix nouveau (différent de auteur.rice et qui serait "invité.e" / "Starring" ?) Comme ça c'est plus facile pour faire lien ? Ou on rajoute l'utilisateur comme auteur. 

<img width="944" alt="Capture d’écran 2022-03-08 à 12 04 57" src="https://user-images.githubusercontent.com/85872944/157226326-ae3e0892-0cf0-470a-b13c-cb3fb21d6b48.png">

**Reply by alemangui** [reply]
C'est possible oui, à définir si on veut qu'un article de blog puisse être lié à zéro ou une cantine, ou bien si un seul article de blog peut concerner plusieurs cantines différentes.

**Reply by gauthierandre** [reply]
Yes! Il y a des articles qui vont toucher qu'une cantine ou plusieurs, je pense quand c'est un gestionnaire de cantine ou cuisine centrale qui témoigne. Des articles réglementaires (je pense à celui sur le décret outre-mer et ça connecte avec le sujet que j'ai posté sur cette question), ils peuvent également toucher plusieurs restaurants suivant le territoire par exemple.
